crates/lovely-core/src/lib.rs

```diff
@@ -223,7 +224,7 @@ pub struct PatchTable {
     loadbuffer: Option<&'static LoadBuffer>,
     targets: HashSet<String>,
     // Unsorted
-    patches: Vec<(Patch, Priority)>,
+    patches: Vec<(Patch, Priority, PathBuf)>,
```

Comment:
At some point we should replace this tuple with a struct. This works for now though.